var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -276,3 +279,8 @@ def check(self):
         #       requires Python with numpy, pyyaml, pandas
         if '~mpi' in self.spec:
             make('tests_in')
+
+    def install(self, spec, prefix):
+        make('install')
+        if '+install-tests' in spec:
+            install_tree('tests', spec.prefix.tests)
```

Comment:
Hi @downloadico FWIW there are also mechanisms to run tests during the build. Do you prefer to have the tests available to run after the installation?

Reply:
@scheibelp Yes, the goal is to have the tests available after the build is complete.